tests/php/SubsiteAdminFunctionalTest.php

```diff
@@ -43,50 +55,53 @@ public function testAnonymousIsForbiddenAdminAccess()
      */
     public function testAdminCanAccessAllSubsites()
     {
-        $member = $this->objFromFixture('Member', 'admin');
-        Session::set("loggedInAs", $member->ID);
-        
+        $member = $this->objFromFixture(Member::class, 'admin');
+        Session::set('loggedInAs', $member->ID);
```

Comment:
`$this->logInAs($member)` or `$this->logInAs('admin')` and remove the previous line